SmartDeviceLink/SDLTextAndGraphicManager.m

```diff
@@ -148,6 +152,8 @@ - (void)sdl_updateWithCompletionHandler:(nullable SDLTextAndGraphicUpdateComplet
     fullShow = [self sdl_assembleShowImages:fullShow];
 
     self.inProgressHandler = handler;
+    const int MAX_ARTWORK_UPLOAD_RETRY_ATTEMPTS = 3;
```

Comment:
This should be near the top and in the standard constant format.

`NSUInteger const SDLMaxArtworkUploadRetryAttempts = 3`

I may also suggest that this be lowered to 1 or 2.